tests/flow/test_union.py

```
@@ -171,4 +171,22 @@ def test09_union_write_read(self):
+    def test10_invalid_pattern_predicate(self):
+        query = """MATCH (p)
+                   WHERE EXISTS((p)-->() )
+                   RETURN 1 AS x
+                   UNION
+                   MATCH (p)
+                   WITH p
+                   RETURN 2 AS x"""
+
+        try:
+            self.graph.query(query)
+            assert(False)
```

Ruff (0.15.21)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)